src/node.h

```diff
@@ -409,17 +409,23 @@ extern "C" NODE_EXTERN void node_module_register(void* mod);
 # define NODE_MODULE_EXPORT __attribute__((visibility("default")))
 #endif
 
+#ifdef NODE_SHARED_MODE
+# define NODE_CTOR_PREFIX
+#else
+# define NODE_CTOR_PREFIX static
+#endif
```

Comment:
One result of removing `static` (if I read the changes right), is that internal module constructors leak into the list of exported symbols.  Or is that mitigated by `-fvisibilty=hidden`?